src/components/Interview/Interview.i18n/ru.json

```diff
@@ -9,6 +9,6 @@
     "Created at": "Создано",
     "Interview sections": "Секции в интервью",
     "CV:": "Резюме:",
-    "Restrict access": "Ограничить доступ",
-    "Comments": "Комментарий"
+    "Comments": "Комментарий",
+    "Restrict / allow access": "Ограничить/выдать доступ"
```

Comment:
May be needs separate this action text by ('Restrict' and 'Allow')?

Reply:
It's the same page for both